### ISSUE TITLE
Incorrect time metrics

### ISSUE THREAD

**Issue by eugene-g** [fsprojects/Paket#2510]
### Description

```
$ sh paket.sh restore
Checking Paket version (downloading latest stable)...
Paket.exe 5.6.4 is up to date.
Paket version 5.6.4
Performance:
 - Disk IO: 1 minute, 59 seconds
 - Runtime: 23 seconds
```

Disk IO is calculated incorrectly here (for comparison: Runtime is very close to wall clock).

### Repro steps

Run paket.

### Expected behavior

Disk IO + Runtime = wall clock

### Actual behavior

Disk IO is incorrect (seems like we sum all the time from parallel threads)

### Known workarounds

Just ignore it.

**Comment by forki** [comment]
@matthid any comments on this?

**Comment by matthid** [comment]
Yes it's a duplicate ;) But I'm on mobile right now to search the original one.

**Comment by forki** [comment]
marked as up-for-grabs

**Comment by eugene-g** [comment]
Original is #2367

**Comment by eriawan** [comment]
@forki 
I think this issue has been fixed by PR #2946, and I have tested it.
Could we close this issue?
